.github/workflows/kvrocks.yaml

```diff
@@ -101,6 +101,9 @@ jobs:
             os: ubuntu-18.04
             with_ninja: --ninja
             without_jemalloc: -DDISABLE_JEMALLOC=ON
+          - name: Ubuntu GCC without luajit
+            os: ubuntu-18.04
+            without_luajit: -DUSE_LUAJIT=OFF
```

Comment:
I think you need add ${{ matrix.without_luajit }} to line 140

Reply:
done https://github.com/apache/incubator-kvrocks/pull/697/commits/d4a6ff9a7a0a38d01f5ea0228714c1cd0d98f644